tests/test-cases/docs-generate.yaml

```
@@ -0,0 +1,84 @@
+      - "generate"
+    expect:
+      diff: []
+      exit_code: 0
+      stderr:
+        - "^Generated docs at .*README.md.*"
+      stdout:
+        - "^$"
+      file_exists:
+        - "README.md"
+      file_contains:
+        - 'something'        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)